operator/cmd/mesh/operator-common.go

```diff
@@ -55,7 +55,7 @@ const (
 // isControllerInstalled reports whether an operator deployment exists in the given namespace.
 func isControllerInstalled(cs kubernetes.Interface, operatorNamespace string, revision string) (bool, error) {
```

Comment:
Also, this is not only used by `operator remove`, I'm not sure if doing it like this will change other commands' behaviors.